### PR TITLE
[KED-2795] Refactor colors based on latest palette

### PR DESCRIPTION
**Pull request by rashidakanchwala**
## Description

Recently, Design did a ticket on revisiting all the colors that were used on Kedro-viz and cleaned up the similar color and created a style guide for colors that we could follow moving forward. 

On the development side, there's a requirement to refactor our colors so they align with the design style guide - https://app.zeplin.io/project/5d5ea7a05efca76a74f7d0ea/screen/60faa9b5f0887815121674f4

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -17,26 +20,24 @@ $color-button-hovered: #005d92;
...
-$color-bg-light-2: #e6e8eb;
+$color-accent-purple: #a000bc;
+$color-bg-light-1: #ffffff;
+$color-bg-light-2: #f8f9fa;
 $color-bg-light-3: #f0f1f3;
-$color-bg-light-4: #f8f9fa;
-$color-bg-light-5: #eeeeee;
+$color-bg-light-4: #e6e8eb;
+$color-bg-light-5: #dadee2;
+$color-bg-dev-light-1: #ccd1d6;
```
Comment (studioswong) [inline src/styles/_variables.scss:29]: It would be worth adding a note to explain where this variable comes from, and what the `dev` indication here means - I presume this is a new color specified here that was used only in the codebase but not specified in the styleguide?

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -17,26 +20,24 @@ $color-button-hovered: #005d92;
...
-$color-bg-light-1: #ccd1d6;
-$color-bg-light-2: #e6e8eb;
+$color-accent-purple: #a000bc;
+$color-bg-light-1: #ffffff;
+$color-bg-light-2: #f8f9fa;
 $color-bg-light-3: #f0f1f3;
-$color-bg-light-4: #f8f9fa;
-$color-bg-light-5: #eeeeee;
+$color-bg-light-4: #e6e8eb;
+$color-bg-light-5: #dadee2;
```
Comment (studioswong) [inline src/styles/_variables.scss:28]: I would recommend adding a note to state that the naming of these colors are adopted and should follow that from the styleguide to prevent them to be updated or overwritten by accident without updating the styleguide in the future.

**Review by studioswong — COMMENTED** [review]
thanks for aligning the colors - overall looks good to me, only left is to add the comment about the colors in the variable to let everyone know that those variables are aligned to our styleguide on zeplin. 

 I recommend for this branch to be tested by design as well to ensure all colors are set up as they intended

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -56,6 +56,7 @@ def count_truthy(elements: List[Any]) -> int:
 
 - Keep UI elements fully keyboard accessible, and aim to support screen-readers where possible
 - Maintain a high level of animation performance, and minimise page-load time
+- We use a specific set of colours which is defined in the style guide docs under theme colours.
```
Comment (studioswong) [inline CONTRIBUTING.md:59]: I would  reword this and also include the document to be more in line with the style of our docs:
_We use the set of colours defined in the Kedro-Viz Color Styleguide for all colour variables and usage. _

File: STYLE_GUIDE.md [hunk 4]
```diff
@@ -119,7 +119,9 @@ color: rgba(black, 0.5);
 
 ### Theme colours
 
-Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme.
+Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme. Below is the style guide for kedro colors 
+
+![Kedro-Viz Color Styleguide](./.github/img/kedro_colours.png)
```
Comment (studioswong) [inline STYLE_GUIDE.md:124]: I would reword this as  `Kedro-Viz has both light and dark themes, and every component must be styled appropriately for each theme. Please refer to Kedro-Viz Color Styleguide for the set of colours for each theme.`

File: src/styles/_variables.scss (hunk 2 above)
Comment (studioswong) [reply src/styles/_variables.scss:33]: and also to add a comment on line 5 to remind whoever that might come across this file the need to check and align with the styleguide :D

**Review by limdauto — APPROVED** [review]
👍 thank you for doing this! This lgtm!

File: STYLE_GUIDE.md (hunk 4 above)
Comment (rashidakanchwala) [reply STYLE_GUIDE.md]: Hi, Susanna -- just for this one I am just saying _Please refer to below colour guide for the set of colors..._ 

Because it's an image below the text and not a link.

**Review by GabrielComymQB — APPROVED** [review]

**Review by studioswong — APPROVED** [review]
